src/components/tables/TableListMultiDisplay/TableListMultiDisplay.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from "react";
+import IReactComponentProps from "../../../common/structures/IReactComponentProps";
+import classnames from "classnames";
+import { TableList } from "../TableList/TableList";
+import isEqual = require("lodash.isequal");
+import * as styles from "../TableList/TableList.sass";
+import { PrimaryButton } from "../../buttons/PrimaryButton/PrimaryButton";
+import { FunctionGeneric } from "../../../common/structures/Generics";
```

Comment:
Not sure if it's a config or editor issue, but would you mind changing these from double quotes to single?